machine/AVAILABLE_DRIVER_PLUGINS.md

```diff
@@ -0,0 +1,47 @@
+<!--[metadata]>
++++
+draft = true
+title = "Machine plugins"
+description = "Machine plugins"
+keywords = ["Docker, documentation, manual, guide, reference, api"]
++++
+<![end-metadata]-->
+
+# Available driver plugins
+
+This document is intended to act as a reference for the available 3rd-party
+driver plugins available in the ecosystem beyond the core Machine drivers.  If
+you have created a Docker Machine driver, we highly encourage you to submit a
+pull request adding the relevant information to the list.  Submitting your
+driver here will allow others to discover it and the core Machine team to keep
+you informed of upstream changes.
+
+**NOTE**: The linked repositories are not maintained by or formally associated
+with Docker Inc.  Use 3rd party plugins at your own risk.
+
+| Name                   | Repository                                                          | Maintainer GitHub Handle                                                                           | Maintainer Email                                             |
```

Comment:
Jekyll is much more picky than Hugo about mixing Markdown and HTML. Consider converting the Markdown to HTML by looking at a preview of this file in Github and copy/pasting the generated HTML here. I'm also not sure why there are brackets around the repo names. Is it to prevent them from becoming automatic links?